src/app/child-dev-project/children/children.service.ts

```diff
@@ -119,14 +123,46 @@ export class ChildrenService {
     );
   }
 
-  getNotesOf(entityId: string, noteProperty: string): Promise<Note[]> {
-    return this.dbIndexing.queryIndexDocs(
+  /**
+   * Query all notes that have been linked to the given other entity.
+   * @param entityId ID (with prefix!) of the related record
+   */
+  async getNotesRelatedTo(entityId: string): Promise<Note[]> {
+    let indirectlyLinkedNotes = [];
```

Comment:
```suggestion
    let legacyLinkedNotes = [];
```
I would not call these *indirectly* because they are linked jus the same way as the `relatedEntities`.

Reply:
true. They are not exactly legacy yet either, if we stick with the current architecture for now ;-)  But yours is more clear :+1: